libraries/kcpassword.rb

```diff
@@ -0,0 +1,36 @@
+module KcpasswordHelpers
```

Comment:
What's the purpose of this top level namespace? It seems we're already mixing these in to `Chef::Recipe` and `Chef::Resource`, so the extra layer of scoping doesn't seem necessary.